ui/src/components/SchemaForm/Widgets/CheckboxesWidget.tsx

```
@@ -0,0 +1,112 @@
+              <Checkbox
+                id={optionId(id, index)}
+                name={id}
+                checked={checked}
+                disabled={disabled || itemDisabled || readonly}
+                autoFocus={autofocus && index === 0}
+                // TODO: Fix this
+                // onChange={_onChange(index)}
+                onBlur={_onBlur}
+                onFocus={_onFocus}
+                aria-describedby={ariaDescribedByIds<T>(id)}
+              />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)